plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -70,7 +70,6 @@
 import static io.trino.testing.TestingConnectorBehavior.SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE;
 import static io.trino.testing.TestingConnectorBehavior.SUPPORTS_CANCELLATION;
 import static io.trino.testing.TestingConnectorBehavior.SUPPORTS_CREATE_TABLE;
-import static io.trino.testing.TestingConnectorBehavior.SUPPORTS_DELETE;
```

Comment:
Separate PR?

Reply:
Was this unused? Why didn't Checkstyle fail?

Maybe I broke it and master is red?

Reply:
Seems like it is actually used: https://github.com/trinodb/trino/blob/master/plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java#L110

Reply:
maven-checks should've failed. Are we not compiling tests in maven-checks. 🤔 

Or maybe @ssheikin 's branch is not in sync with trinodb/master and hence git is getting confused?

Reply:
Good catch. This was the automatic change by IDE. 

Maven checks are green here and on master.

cc @ksobolew

Reply:
I think the import is not necessary when the enum constant is used in a case label. That's just how `switch` works. And it's the only use of `SUPPORTS_DELETE`, so.

Reply:
> Are we not compiling tests in maven-checks

There's an old Maven trap in that the `-Dmaven.test.skip` skips more than tests - it skips compilation of tests as well. I've had some cases at my previous previous previous employer where people committed code which did not compile because they were running Maven with this switch and they didn't notice that their refactoring broke the tests. But that's not the case here, fortunately :)

Reply:
so seems like maven-checks just cannot distinguish such situation, and therefore don't fail on master.

Reply:
We don't use `-Dmaven.test.skip` anywhere thankfully due to this reason.

I didn't know that about the `switch` statement. Today I learnt. Thanks @ksobolew .